CS342MemberViews/Sources/BrookeKrajancich/ProfilePanel.swift

```diff
@@ -0,0 +1,73 @@
+// swift-tools-version: 5.7
+
+//
+// This source file is part of the CS342 2023 Application project
+//
+// SPDX-FileCopyrightText: 2023 Stanford University
+//
+// SPDX-License-Identifier: MIT
+//
+
+import SwiftUI
+
+struct ProfilePanel: View {
+    private var profileImage: Image {
```

Comment:
I do generally advise that you group computed properties like `profileView` and `backgroundImage` below any stored properties (`backgroundColor` and `highlightColor`) in your case.